docs/resources/metric_alert.md

```diff
@@ -35,6 +35,18 @@ resource "sentry_metric_alert" "main" {
     threshold_type  = 0
   }
 
+   trigger {
+    action {
```

Comment:
The indentation here looks a bit funny. I think the indentation level of `action` needs one more space